src/common.fypp

```diff
@@ -27,6 +27,15 @@
 #! Collected (kind, type) tuples for integer types
 #:set INT_KINDS_TYPES = list(zip(INT_KINDS, INT_TYPES))
 
+#! Integer kinds to be considered during templating
```

Comment:
My GitHub UI is broken so I can't suggest it directly. In lines 30, 33, and 36 integer can replaced with logical in the comments.